app.py

```diff
@@ -139,10 +139,13 @@ def call_api():
                 try:
                     request = urllib2.Request(_config['api_endpoint'])
                     request.add_header("Authorization", "Bearer %s" % user.access_token)
+                    request.add_header("Accept", 'application/jwt, application/json;q=0.9')
```

Comment:
NGINX would segfault without this. @anestos fixed that, but the fix isn't merged yet. In any case, this is good to add.

Reply:
This is the api-call, accept: jwt seems out of place?

Reply:
Indeed. I think it was because of that NGINX bug, so I coped it from somewhere else. Should just be `request.add_header("Accept", 'application/json')`